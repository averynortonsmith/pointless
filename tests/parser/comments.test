
------------
-- comment
import "foo" as --bar
 bar
-- foo
x -- comment
 = 123