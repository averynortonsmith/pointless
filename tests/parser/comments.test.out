(Program 
 null
 (Import 
  (String "foo")
  (Name "bar"))
 (Def 
  (Name "x")
  (Number 123)))
